src/Stratis.Bitcoin.IntegrationTests.Common/Stratis.Bitcoin.IntegrationTests.Common.csproj

```diff
@@ -95,6 +95,9 @@
     <None Update="ReadyData\StratisTest150NoWallet.zip">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
+    <None Update="ReadyData\StratisXOver15K.zip">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```

Comment:
Perhaps change to `Copy Always` to keep ion line with rest?

Reply:
Why? It will just increase rebuild time

Reply:
I don't care either way, but lets keep things consistent. Either change this to Copy Always or change the others to `PreserveNewest`.